api-spec/protobuf/tdex/v2/trade.proto

```diff
@@ -107,7 +112,7 @@ message ProposeTradeRequest {
 message ProposeTradeResponse {
   SwapAccept swap_accept = 1;
   SwapFail swap_fail = 2;
-  uint64 expiry_time_unix = 3;
+  uint64 expiry_time_unix = 3 [(google.api.field_behavior) = REQUIRED];
```

Comment:
not needed since 0 is default for `uint64`s. Also, this value is != 0 only if SwapAccept is not null.